docs/release_notes/2024/oct2024.md

```diff
@@ -0,0 +1,116 @@
+
+  - Infer the variant schema
+  - Configure the parsing limit for inferring the column structure
+  - Use a nested column inside of the Visual Expression Builder
+
+  For more information, see [Variant schema](/docs/SQL/development/visual-editor/variant-schema.md).
+
+- **Flatten schema Gem**: When processing raw data it can be useful to flatten complex data types like `Struct`s and `Array`s into simpler, flatter schemas. This Gem builds upon the variant type support by allowing you to preserve all schemas, and not just the first one.
+
+  ![Input schema](./img/oct-flatten_input.png)
+
+  FlattenSchema works on Snowflake sources that have nested columns that you'd like to extract into a flat schema.
+
+  For more information, see [Flatten schema](/docs/SQL/gems/transformations/flattenschema.md).
+
+- **Deduplicate Gem**: You can use the deduplicate Gem to remove rows with duplicate values of specified columns. There are four **Row to keep** options that you can use in your deduplicate Gem.
+
+  ![Deduplicate Row to keep](./img/oct-deduplicate_row_to_keep.png)
+
+  For more information, see [Deduplicate](/docs/SQL/gems/transformations/deduplicate.md).
+
+- **Gem builder**: You can add custom Gems to your SQL projects using the Gem builder. You can create custom source, target, and transformation Gems, and then publish them for your team to use.
+
+  Our SQL Gem builder supports Databricks and Snowflake SQL.
+
+  ![Gem builder customize interface](./img/oct-gem-builder-interface.png)
+
+  For more information, see [Gem builder](/docs/SQL/extensibility/gem-builder/gem-builder.md).
+
+### Minor Improvements {#MinorImprovements34}
+
+- **Code diff on merge screen**: You can view the code changes of your commit when you start the process to commit changes. This gives you granular change visibility so that you can understand the detailed changes being made to your Pipelines, Models, datasets, and other entities.
+
+  ![View Git code changes](./img/oct-git-code-changes.png)
+
+  You can also use manual merge to resolve conflicts. This provides you with simple, yet effective ways to resolve merge conflicts for granular changes. For SQL, you can also toggle on **Code Changes** to view and edit the code directly on the files.
+
+  For more information, see the [Git docs](/docs/metadata/git/git.md).
+
+- **Voice interface in Copilot conversations**: You can now use audio to interact with Data Copilot. Copilot will transcribe your request and read aloud the text response shown in the chat.
+
+![Voice interface in copilot conversations](./img/oct-voice-interface-copilot.png)
+
+You must use Google Chrome to use the voice interface.
+
+- **Modify Gems in Copilot conversations**: We've improved the modify Gem experience when using Data Copilot. When making your requests, you don't have to select the Gems that you want to change. You can simply tell Copilot in your request or let it figure out which Gems to update.
```

Comment:
We can combine these features and put them in AI copilot section instead of minor enhancements:

1. Voice interface in copilot
2. Enhanced gem modification in AI copilot
3. Prompt suggestions in copilot chat - https://app.asana.com/0/1205667531654291/1207931483053731/f (this is missing from notes)